Amigolden/src/environments/environment.ts

```diff
@@ -4,21 +4,24 @@
 
 export const environment = {
   production: false,
-  odataMetadataurl: 'https://localhost:44377/OData/$metadata',
-  baseUrl: 'https://localhost:44377/',
-  apiUrl: 'https://localhost:44377/api',
+  odataMetadataurl: "https://localhost:44377/OData/$metadata",
+  baseUrl: "https://localhost:44377/",
+  // apiUrl: 'https://localhost:44377/api',
+  apiUrl: "https://www.amigolden.com/api/api",
   authentication: {
-    userTokenName: 'UT-8675309'
+    userTokenName: "UT-8675309",
   },
   googleMaps: {
-    apiKey: 'AIzaSyCXBaTUeO21FjzLJnghunS7CkDnLcLnYm4'
+    // apiKey: "AIzaSyCXBaTUeO21FjzLJnghunS7CkDnLcLnYm4",
+    // apiKey: "AIzaSyD7psaOzyFhNTJJaADO93nRc2xJnsdl_CY",
+    apiKey: "AIzaSyAIgQez_AR0WN2_Kwx_SimJkOCtHQyzMdE",
```

Comment:
please avoid commiting test api keys since they're not restircted by domain, this could cause issues